lib/country_data.rb

```diff
@@ -128,6 +129,10 @@ def popolo
       def statement_count
         json_with_count.last
       end
+
+      def type
+        popolo[:organizations].find { |o| o[:classification] == 'legislature' }[:type] || ''
```

Comment:
I think it's probably worth removing the duplication with `lname` here by factoring out a `legislature` method that both call.